mp/src/game/client/neo/ui/neo_ui.cpp

```diff
@@ -279,9 +284,20 @@ void EndSection()
 
 	// Scroll handling
 	const int iRowsInScreen = g_pCtx->dPanel.tall / g_pCtx->iRowTall;
-	if (g_pCtx->eMode == MODE_MOUSEWHEELED && g_pCtx->bMouseInPanel)
+	const bool bHasScroll = (g_pCtx->iPartitionY > iRowsInScreen);
+	vgui::IntRect rectScrollArea = (bHasScroll) ?
```

Comment:
Sorry for nitpicking, but why parentheses?

Reply:
I generally have a preference on wrapping ternary conditional's condition with parentheses regardless if required or not.